app/assets/stylesheets/container_providers_dashboard.css

```diff
@@ -0,0 +1,45 @@
+
+.miq-fixed-body {
+    padding-top: 125px;
+}
+
+.miq-body {
+    background: #f5f5f5;
+}
+
+.miq-dashboard-view .card-pf-aggregate-status .card-pf-title .pficon, .miq-dashboard-view .card-pf-aggregate-status .fa, .miq-objects-view .card-pf-aggregate-status .card-pf-title .pficon, .miq-objects-view .card-pf-aggregate-status .fa {
+    color: #1186C1 !important;
+}
+
+.miq-dashboard-view .data-list-pf .list-group-item.active .pficon, .miq-dashboard-view .data-list-pf .list-group-item.active:hover .pficon, .miq-dashboard-view .data-list-pf .list-group-item.active:focus .pficon, .miq-objects-view .data-list-pf .list-group-item.active .pficon, .miq-objects-view .data-list-pf .list-group-item.active:hover .pficon, .miq-objects-view .data-list-pf .list-group-item.active:focus .pficon {
+    color: #1186C1;
+}
+
+.miq-dashboard-view .data-list-pf .list-group-item .pficon, .miq-objects-view .data-list-pf .list-group-item .pficon {
+    font-size: 18px;
+}
+
+.miq-dashboard-view .tiles-view-pf .tile.active .pficon, .miq-dashboard-view .tiles-view-pf .tile.active:hover .pficon, .miq-dashboard-view .tiles-view-pf .tile.active:focus .pficon, .miq-objects-view .tiles-view-pf .tile.active .pficon, .miq-objects-view .tiles-view-pf .tile.active:hover .pficon, .miq-objects-view .tiles-view-pf .tile.active:focus .pficon {
+    color: #1186C1;
+}
+
+.miq-dashboard-view .card-pf .loading, .miq-objects-view .card-pf .loading {
+    margin: 10px auto;
+}
+
+.miq-dashboard-view .pficon-kubernetes::before, .miq-objects-view .pficon-kubernetes::before {
```

Comment:
> @abonas if theres an icon, the code should be easy and can go in this patch.

there is. check the patch I mentioned. my concern is how 3 icons will look in that tile - it will require style  changes to accommodate all 3.